### PR TITLE
privacy update for signup.html

### PR DESCRIPTION
**Pull request by prycrane**
**Design document**

- https://docs.google.com/document/d/12tQKFecp58ES4KdYVhh00D4vQgbJQObYqGA6kWJR2n4/edit

### CODE REVIEW

**Comment by prycrane** [comment]
- @delisma - please do a technical review of this pull request
- @donohuech - please do a content review of this pull request (question - should the acts and directives be italicized in English?)

**Thanks!**

**Review by delisma — CHANGES_REQUESTED** [review]
I think we're missing the mmodifications in `fr/paages/inscrire.html`

**Comment by prycrane** [comment]
@delisma - page updated.  Good catch.  Thanks!

File: _data/locales/signup.yml [hunk 1]
```diff
@@ -23,14 +23,14 @@ privacy-notice:
   en: Privacy notice
   fr: Énoncé de confidentialité
 privacy-notice1:
-  en: Our newsletter feature uses <a href="https://mailchimp.com/">Mailchimp</a>, a third-party application. Sign-up is voluntary and subject to the <a href="https://mailchimp.com/legal/terms/">Terms of Use</a> for the Mailchimp platform.  When you sign up, your email address and language preference (English or French) is collected. No other personal information is collected.
-  fr: Notre fonction Bulletin de nouvelles fait appel &agrave; <a href="https://mailchimp.com/">Mailchimp</a>, une application tierce. L'adh&eacute;sion est libre et est assujettie aux <a href="https://mailchimp.com/legal/terms/">Conditions d'utilisation</a> associ&eacute;es &agrave; la plateforme Mailchimp. Au moment de l'adh&eacute;sion, le syst&egrave;me enregistre votre adresse &eacute;lectronique et votre langue de pr&eacute;f&eacute;rence (anglais ou fran&ccedil;ais). Aucun autre renseignement personnel n'est recueilli.
```
Comment (donohuech) [inline _data/locales/signup.yml:27]: Can you please change the Mailchimp.com link to the French version of the link which is: https://mailchimp.com/fr

**Review by donohuech — CHANGES_REQUESTED** [review]

File: _data/locales/signup.yml (hunk 1 above)
Comment (prycrane) [reply _data/locales/signup.yml:27]: All fixed

**Review by delisma — APPROVED** [review]
LGTM 🚀

**Review by donohuech — APPROVED** [review]

**Pull request merged**